src/commands.rs

```diff
@@ -41,7 +46,9 @@ pub async fn jump(
         // TODO: log failure to create file
         let _ = uri.to_file_path().map(|path| {
             path.parent().map(|parent| std::fs::create_dir_all(parent));
-            File::create_new(path.as_path().to_owned())
+            if !path.exists() {
```

Comment:
heads up, the decision to use `create_new` was deliberate--using `!path.exists` is subject to a TOCTOU issue, where a file created between `exists` and `create` will be erased (could occur with an autosave, for example)